kratos/python/add_geometrical_utilities_to_python.cpp

```diff
@@ -0,0 +1,352 @@
+//
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define_python.h"
+#include "processes/process.h"
+#include "python/add_geometrical_utilities_to_python.h"
+
+//Geometrical utilities
+#include "utilities/normal_calculation_utils.h"
+#include "utilities/body_normal_calculation_utils.h"
+#include "utilities/body_distance_calculation_utils.h"
+#include "utilities/signed_distance_calculation_utils.h"
+// #include "utilities/signed_distance_calculator_bin_based.h"
+#include "utilities/parallel_levelset_distance_calculator.h"
+#include "utilities/brute_force_point_locator.h"
+#include "utilities/deflation_utils.h"
```

Comment:
I think this should go to the other utilities.

Reply:
You are right. Changed.